buildSrc/src/main/kotlin/kotlin/org/jetbrains/kotlin/gradle/targets/jvm/tasks/KotlinJvmTest.kt

```diff
@@ -1,50 +0,0 @@
-// Copied from https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/jvm/tasks/KotlinJvmTest.kt
```

Comment:
Didn't worked?

Reply:
it works for JVM, but I started to get similar issue for `linuxX64Tests`
Decided to rollback gradle

Reply:
> it works for JVM, but I started to get similar issue for `linuxX64Tests` 
unpleasant